libnmstate/nm/applier.py

```diff
@@ -95,6 +95,9 @@ def prepare_edited_ifaces_configuration(ifaces_desired_state):
             con_profiles.append(cur_con_profile)
         else:
             # Missing connection, attempting to create a new one.
+            device.delete_device_inactive_profiles(
+                iface_desired_state[Interface.NAME]
```

Comment:
This is first used on line 82 so it merits its own variable and just re-using it everywhere.

Reply:
Done!